WordPress/Classes/ViewRelated/Reader/Detail/ReaderDetailViewController.swift

```diff
@@ -444,13 +450,10 @@ class ReaderDetailViewController: UIViewController, ReaderDetailView {
 
     /// Apply view styles
     @MainActor private func applyStyles() {
-        guard let readableGuide = webView.superview?.readableContentGuide else {
-            return
-        }
-
+        webView.pinEdges(.horizontal, to: view, insets: UIEdgeInsets(.horizontal, 16), priority: .init(950))
         NSLayoutConstraint.activate([
-            webView.rightAnchor.constraint(equalTo: readableGuide.rightAnchor, constant: -Constants.margin),
-            webView.leftAnchor.constraint(equalTo: readableGuide.leftAnchor, constant: Constants.margin)
+            webView.centerXAnchor.constraint(equalTo: view.centerXAnchor),
```

Comment:
The `pinEdges` call adds leading and trailing constraints, which would implicitly pin `webView.center.x` to `view.center.x`. So, I think this constraint is not necessary?